### PR TITLE
Add docs for how to release the registry

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,4 +175,20 @@ If you want to run the most recent registry, run the master tag.
 
 ### Healthcheck
 
-For Docker / Kubernetes the `/` endpoint can be queried. As soon as `/` returns a 200, the service is ready.
+For Docker / Kubernetes the `/health` endpoint can be queried. As soon as `/health` returns a 200, the service is ready.
+
+## Release
+
+New versions of the package registry need to be released from time to time. The following steps should be followed to create
+a new release:
```
Comment (ycombinator) [inline README.md:183]: Nit: unnecessary newline.

File: README.md [hunk 2]
```diff
@@ -177,4 +175,20 @@ If you want to run the most recent registry, run the master tag.
...
+
+1. Create a new branch with the changes to be done for the release
+2. Update the changelog by putting in a line for the release, remove all not needed section and put in a new Unrelease section. Don't forget to update the links to the diffs.
```
Comment (ycombinator) [inline README.md:186]: Typo: should be Unreleased?

File: README.md [hunk 3]
```diff
@@ -177,4 +175,20 @@ If you want to run the most recent registry, run the master tag.
...
+3. Update the registry version in the `main.go` file to be the same version as the release is planned and update the generated files with `go test . -generate`.
+4. Open a pull request and get it merged
+5. Tag the new release by creating a new release in Github, put in the changelog in the release
+6. Update the main.go to increase the version number to the version of the potential next release version.
+
+CI automatically creates a new Docker image which will be available under `docker.elastic.co/package-registry/package-registry:vA.B.C` a few minutes after creating the tag.
+
+As a new registry is normally released to bring new features to the package-storage, follow the docs in the [Package Storage](https://github.com/elastic/package-storage#update-package-registry-for-a-distribution) repository on how to update the distributions.
```
Comment (ycombinator) [inline README.md:194]: This is good here fo not but would it be possible in the future to have the CI job that updates the package-registry docker image to automatically kick off the package-storage jobs downstream?

File: README.md (hunk 1 above)
Comment (ruflin) [reply README.md:183]: removed.

File: README.md (hunk 3 above)
Comment (ruflin) [reply README.md:193]: Maybe. I could see this at least for snapshot and if things go well, we roll it out to staging and production too.

**Pull request merged**